apps/example/src/domain/payer.ts

```diff
@@ -5,9 +5,11 @@ import { equals } from "ramda"
 import * as Config from "../config/equinox.js"
 
 export namespace Stream {
-  export const CATEGORY = "Payer"
+  export const category = "Payer"
   export const streamId = StreamId.gen(PayerId.toString)
```

Comment:
`id` is not better, but it would be nice to find a name that nods to the streamid and decodeid being related
createId vs decodeId
genId vs decodeId
streamId vs decodeStreamId ?